src/droplet.F

```diff
@@ -489,7 +488,7 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
 
       !Project the feedback onto the grid, using ORIGINAL location
       call project_feedback(iflag,jflag,kflag,x3d0,y3d0,z3d0,sig3d0, &
-                            rhop0,taup0,rp0,tval,uten,vten,wten,qten,thten, &
+                            rhop0,taup0,rp0,tval,uten,vten,wten,qten,thten,dten, &
```

Comment:
So `uten`, `vten`, `wten`, `qten` and `thten` are still used in the **project_feedback** subroutine. Should them be replaced by `dten` as well?

Reply:
`dten` is a new stats variable that was created to get a 3D grid of the droplet numbers. So this is just adding to it, not replacing any of the others.

Reply:
I was just about to revise my comment above about renaming the variables inside project_feedback. We could simply remove all references to `uten`, `vten`, `wten`, `qten`, `thten`, and `dten` and just use `dpten` as appropriate. Would that work?

Reply:
I think this will be great. I also suggest that we should replace all those `Xten` variables by `dpten` in CM1 to avoid confusion and potential bugs.

Reply:
Wait, but that wouldn't work. In cm1.F and solve3.F, they all have different sizes. `dpten` would only be referring to the droplet feedback. That way, any reference to `Xten` inside cm1.F or solve3.F is coming from some other tendency, NOT the droplets.

Reply:
So you mean `uten`, `vten`, etc in cm1.F, solve3.F and droplet.F are actually different variables and not related to each other?

Reply:
In cm1.F, there are "ten" terms for all of the variables with some governing equation (`uten`, `vten`, ... `qvten`, `thten`, `ppten`, etc). These are supposed to be general and can account for any source/sink in any of the associated governing equations. Due to grid staggering, they don't all have the same size (hence the ie+1, je+1, etc), and so there is no convenient way to consolidate them all into a single "ten" array. 

So these are what are used consistently throughout cm1.F and solve3.F. The plan was to do the same thing with the droplets (hence why they were all passed in individually at first). But George configured the droplet feedbacks so that the "ten" arrays associated with droplets could all have the same size (again, I need to confirm with him how exactly he did this). This allowed him to consolidate them into a single `dpten` array. So physically, the various elements have the exact same meaning as `uten`, `vten`, etc. in cm1.F and solve3.F -- but specific to the droplets, these were computed in a way that allows them to be joined together into a single array. Hope that makes sense?

Reply:
No, these don't need to be mapped back. They get used immediately after droplet_driver inside solve3.F, inside the `IF ( two_way_coupling )` block. Here the droplet tendencies are "felt" by the velocity, temperature, and humidity directly by `dpten`, and afterwards `dpten` is no longer needed (except for stats).